src/pages/CastBallotPage.tsx

```diff
@@ -55,8 +55,14 @@ const VoteInstructionsPage = () => {
             <Text narrow>
               If you find a mistake, ask a poll worker for help.
             </Text>
+            <div aria-label="Press the down arrow to continue, then" />
```

Comment:
then…? Does this make sense?

Reply:
It works well because it reads the next button instructions too. It's the least disruptive way to make this work without restructuring the HTML